packages/read-model/.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: php
+
+php:
+    - 7.1
+    - 7.2
+    - 7.3
+
+cache:
+    directories:
+        - ~/.composer/cache
+
+before_install:
+    - echo "extension = redis.so" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini
+
+install:
+    - composer install
```

Comment:
in packages.json i see that it is dependent on shopsys/framework, please do not forget to set COMPOSER_MEMORY_LIMIT=-1 for the travis job.

Reply:
@vitek-rostislav please fix it. By the thumbsup I assumed it is fixed

Reply:
@vitek-rostislav ping

Reply:
done in Travis configuration - https://travis-ci.org/shopsys/read-model/settings